a653rs-router/src/ports.rs

```diff
@@ -1,201 +1,20 @@
 use a653rs::{bindings::*, prelude::*};
-use core::{fmt::Display, marker::PhantomData, str::FromStr, time::Duration};
+use core::{fmt::Display, time::Duration};
 
 use crate::{
-    config::{QueuingInCfg, QueuingOutCfg, SamplingInCfg, SamplingOutCfg},
     network::PayloadSize,
-    prelude::*,
     router::{RouterInput, RouterOutput},
 };
 
 #[derive(Debug)]
 pub(crate) enum Port<H: ApexQueuingPortP4 + ApexSamplingPortP4> {
```

Comment:
We may take the opportunity and move this to router.rs and make it private. It is only used for storage inside `RouterResources` and not meant to be part of a public API.